### DISCUSSION TITLE
Config: Renovatebot should only create MR/Suggestions for patch, minor, digest

### DISCUSSION THREAD

**Discussion by werksarztzentrum** [renovatebot/renovate#22584] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

[35.107.0](https://github.com/renovatebot/renovate/releases/tag/35.107.0)

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I want only MR´s from Renovatebot for patch, minor und digest updates. But it also creates MRs for major updates.

### What you tried so far.

I tried several configurations from the internet, but it does not stop creating PRs for major package updates. Also i deletes every renovate mr, the issue & created branches and restart the renovate job. Also i tried to find similar questions in the discussion section, issue section or on stackoverflow (and google)

### Relevant debug logs

<details><summary>Current Renovate.json</summary>

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "labels": ["dependencies"],
  "packageRules": [
    {
      "matchDepTypes": [ "devDependencies", "require-dev" ],
      "matchUpdateTypes": [ "patch", "minor", "digest"]
    },
    {
      "matchDepTypes": [ "dependencies" ],
      "matchUpdateTypes": [ "patch", "minor", "digest"]
    }
  ]
}

```

</details>

### Example MR-Output
![image](https://github.com/renovatebot/renovate/assets/6212996/a562b05e-c885-4326-9cf9-8bd7f386ef2b)

**Comment by rarkins** [comment, marked as answer]
Your packageRules _do nothing_. They match, but then don't take action. You need to do something like matchUpdateTypes=major + enabled=false
